uwblueprint/community-fridge-kw
# build: add commitlint and husky dependencies

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Configure-commit-formatting-dependencies-cf68250f799a4c928fdf9ea4e9ae66a2)



## Implementation description
* Added [depcheck ](https://www.npmjs.com/package/depcheck) and [patch-package](https://www.npmjs.com/package/patch-package) to frontend
* Added husky and commitlint to root for[ conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting



## Steps to test
1. Pull changes
2. Make an edit 
3. Try saving commit without format and should be error 
i.e. 
<img width="631" alt="Screen Shot 2021-09-18 at 2 03 28 PM" src="https://user-images.githubusercontent.com/19617248/133898294-b436a848-5115-41ac-a58d-d2e595152d87.png">




## What should reviewers focus on?
* If the deps make sense 


## Checklist
